starter-code/index.html

```diff
@@ -3,40 +3,59 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+  <div class = "header">
+    <img src="images/spotify-logo.png" alt="spotify-logo" id="spotify-logo">
+    <ul class = "nav">
+      <li>Premium</li>
+      <li>Discover</li>
+      <li>Help</li>
+      <li>Download</li>
+    </ul>
+
+  </div>
+  <div class = "content">
+    <div class = "music-for-everyone">
+        <h2>Music for everyone.</h2>
+        <p>Spotify is now free on mobile, tablet and computer.
+            Listen to the right music, wherever you are.</p>
+      </div>
+
+    <div class="whats-on-spotify">
+        <h3>What’s on Spotify?</h3>
+
+        <p id="bold-green">Millions of Songs</p>
+        <p>There are millions of
+            songs on Spotify</p>
+
+
+        <p id="bold-green">HD Music</p>
+        <p>Listen to music as if you
+            were listening live</p>
+
+        <p id="bold-green">Stream Everywhere</p>
+        <p>Stream music on your
+            smartphone, tablet or
+            computer</p></div>
+
+    <div class="yeezy">
```

Comment:
Same here, you can create a big div that will be the green block, and then two div (one for the left content & one for the image), it's going to be easier for you after to play with those element : 

```html 

<div id="green-container">
      <div id="green-container-left">
        <h2>It’s as yeezy as Kanye West.</h2>
        <hr>
        <ul>
          <li>
            <h3>Search</h3>
            <p> Know what you want to listen to?
              <br>Just search and hit play. </p>
          </li>
          <li>
            <h3>Browse</h3>
            <p> Check out the latest charts,
              <br> brand new releases and great
              <br>playlists for right now. </p>
          </li>
          <li>
            <h3>Discover</h3>
            <p> Enjoy new music every Monday
              <br>with your own personal playlist.
              <br> Or sit back and enjoy Radio.
            </p>
          </li>
        </ul>
      </div>
      <div id="green-container-img">
        <img src="images/spotify-app.jpg">
      </div>
    </div>

```